attiny.go

```diff
@@ -146,6 +187,10 @@ func (a *attiny) UpdateWifiState() error {
 }
 
 func (a *attiny) checkIsOnBattery() (bool, error) {
+	var requiredVersion uint8 = 4
+	if a.version < requiredVersion {
+		return false, a.wrongVersionError("checkIsOnBattery", requiredVersion)
+	}
```

Comment:
It would be cleaner to have a method which does the required version check and returns wrongVersionError if there's a problem.